packages/fx-core/resource/strings.json

```
@@ -25,7 +25,7 @@
     "LocalDebugTenantConfirmNotice": "You already run local debug for the Teams app in another tenant '%s'. You can sign out %s and sign in with the matched M365 account to re-use the previous local debug resource. If you really want to switch tenant for local debug, you can delete the localSettings.json file and re-run local debug.",
     "MigrateSuccessNotice": "Your project is now initialized with Teams Toolkit! You can debug your application by pressing F5. If you wish to host your application in Azure, we recommend you to re-create your project directly using the latest Teams Toolkit.",
     "MigrationToArmAndMultiEnvMessage": "Teams Toolkit will migrate your project to support the latest features. This process will change the contents of the '.fx' folder. Backup this folder or put your project in source control if you want to revert your project back below Teams Toolkit 2.7.0.",
-    "MigrationToArmAndMultiEnvSuccessMessage": "Migration Success! please reload VSCode and run provision command before executing other commands, otherwise the commands may fail. Provision may create new resource, click the link for more information",
+    "MigrationToArmAndMultiEnvSuccessMessage": "Migration Succeeds! We have updated your project with latest Teams Toolkit features. If your project contains Bot or Messsaging Extension capability, please provision your project again. Click on Read more to learn why this is needed.",
```

How about upgrade instead of migration? 'Upgrade' is more acceptable to users.  @MuyangAmigo @wenytang-ms  If you like, I can update others message

Also added 'please reload VSCode' to remind user to click 'Reload' button.  If not reload VSCode, TreeView is not available. 

Added 'Click on Read more to learn why this is needed or how to restore your project' to users who want to restore their project.

```suggestion
    "MigrationToArmAndMultiEnvSuccessMessage": "Upgrade Succeeds! We have updated your project with latest Teams Toolkit features and please reload VSCode. If your project contains Bot or Messsaging Extension capability, please provision your project again. Click on Read more to learn why this is needed or how to restore your project"
```